src/binaryen_stubs_modules.c

```diff
@@ -125,6 +132,19 @@ caml_binaryen_module_write_text(value _module) {
   CAMLreturn(text);
 }
 
+// There is something weird with this function that causes a bunch of newlines to
+// be printed on stdout when calling it. Not sure if that's a bug in Binaryen.
```

Comment:
@ospencer can you look at this a little bit and see if there is anything obvious that might cause it?

Reply:
Nothing in this code would really cause that. Looking at the implementation though, it seems like they print out newlines even on nodes that wouldn't otherwise print anything: https://github.com/WebAssembly/binaryen/blob/3b6744c5195584bd18a7a9cec559755dbfbe6968/src/passes/Print.cpp#L3611